tests/ice40/run-test.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+for x in *.v; do
+  echo "Running $x.."
+  ../../yosys -q -s ${x%.v}.ys -l ./temp/${x%.v}.log $x
```

Comment:
I think it might be failing because the `temp` dir doesn't exist.

Can you change the script to be more similar to this:
https://github.com/YosysHQ/yosys/blob/71dd412ac55860cbf51d91d26088515978f70116/tests/various/run-test.sh

(but without the `*.sh` loop?)

Reply:
done